docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -0,0 +1,398 @@
+  DRIVER_CONFIG=./config.json
+  RELAY_ENDPOINT=<relay-hostname>:<relay-port>
+  NETWORK_NAME=<network-name>
+  DRIVER_PORT=<driver-server-port>
+  INTEROP_CHAINCODE=<interop-chaincode-name>
+  EXTERNAL_NETWORK=<docker-bridge-network>
+  DOCKER_IMAGE_NAME=weaver-fabric-driver
+  DOCKER_TAG=1.2.1
+  DOCKER_REGISTRY=ghcr.io/hyperledger-labs
+  ```
+  `<path_to_connection_profile>` should point to the path of a connection profile you generated in the "Pre-Configuration" section. A Fabric driver obtains client credentials from one of the organizations in your network, so pick an organization and point to the right connection profile.
+
+  The `DRIVER_CONFIG` variable should point to the `config.json` (you can name this whatever you wish) specified below.
+
+  `<relay-hostname>` should be set to the hostname of the relay server machine and `<relay-port>` should match the `port` value in the relay's `config.toml` (see above).
+
+  The `NETWORK_NAME` variable should be a unique ID referring to the Fabric network. It will be used to distinguish container names and wallet paths. (This setting is relevant in situations where a driver is used to query multiple network channels.)
+
+  The `DRIVER_PORT` variable should be set to the port this driver will listen on.
+
+  The `INTEROP_CHAINCODE` variable should be set to the ID of the Fabric Interop Chaincode installed on your Fabric network channel.
+
+  The `EXTERNAL_NETWORK` variable should be set to the [name](https://docs.docker.com/compose/networking/) of your Fabric network.
+- `config.json`: This contains settings used to connect to a CA of a Fabric network organization and enroll a client. A sample is given below:
+  ```
+  {
+      "admin":{
+          "name":"admin",
+          "secret":"adminpw"
+      },
+      "relay": {
+          "name":"relay",
+          "affiliation":"<affiliation>",
+          "role": "client",
+          "attrs": [{ "name": "relay", "value": "true", "ecert": true }]
+      },
+      "mspId":"<msp-id>",
+      "caUrl":"<ca-service-endpoint>"
+  }
+  ```
+  As in the `.env` configuration, you should pick an organization for the driver to associate with. The `admin` section specifies the registrar name and password (this should be familiar to any Fabric network administrator) used to enroll clients. Default values of `admin` and `adminpw` are specified above as examples, which you should replace with the right values configured in your network organization's CA.
+
+  `<affiliation>` should be what's specified in your organization's Fabric CA server configuration. The default is `org1.department1`, but you should look up the appropriate value from the CA server's configuration file.
+
+  `<msp-id>` should be set to the (or an) MSP ID of the selected organization.
+
+  `<ca-service-endpoint>` should be set to the CA server's endpoint. If you launched your CA server as a container from a docker-compose file, this should be set to the container's service name.
```

Comment:
Can it be mentioned here that if connection-profile contains CA details, then this field can be left empty.

Reply:
Fixed.